### PR TITLE
docs(SvgIcon): remove page in production

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -31,7 +31,9 @@ const pkg = require('@fluentui/react-northstar/package.json');
 const componentMenu: ComponentMenuItem[] = _.sortBy(componentInfoContext.parents, 'displayName');
 const behaviorMenu: ComponentMenuItem[] = require('../../behaviorMenu');
 
-const componentsBlackList = ['Debug', 'Design', 'Datepicker'];
+const PROD = process.env.NODE_ENV === 'production';
```
Comment (layershifter) [inline packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx:34]: There is no sense to embed this condition. I am usually using search by `process.env.NODE_ENV` if there is something conditional

Comment (assuncaocharles) [reply packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx:34]: We were reusing this condition in 2 places in this file, I would be introducing the 3rd. Should it always be "in place"?

File: packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -228,7 +228,7 @@ const baseTreeItems: TreeProps['items'] = [
           to: '/component-architecture',
         },
       },
-      ...(process.env.NODE_ENV !== 'production'
+      ...(process.env.NODE_ENV === 'production'
```
Comment (mnajdova) [inline packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx:231]: Why did we change this condition??

**Review by mnajdova — APPROVED** [review]
LGTM thanks for doing this

**Review by layershifter — APPROVED** [review]

**Pull request merged**